packages/otelbin/src/app/og/[id]/route.tsx

```diff
@@ -0,0 +1,64 @@
+// SPDX-FileCopyrightText: 2023 Dash0 Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ImageResponse, type NextRequest } from "next/server";
+import { calcNodes } from "~/components/react-flow/useClientNodes";
+import ParentsNode from "../ParentsNode";
+import { Redis } from "@upstash/redis/nodejs";
+import { getShortLinkPersistenceKey } from "~/lib/shortLink";
+import { calcScale, parseUrlFragment } from "~/lib/metadataUtils";
+import type { IConfig } from "~/components/react-flow/dataType";
+
+export const runtime = "edge";
+
+const redis = Redis.fromEnv();
+const edgeWidth = 80;
+
+export async function GET(request: NextRequest) {
```

Comment:
**Suggestion:** Move the route to `social-preview/[id]/img/route.ts` to put related logic closer together.

Reply:
New route implemented.